src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -117,7 +119,10 @@ class VanillaUmiConsensusCaller
 
   // Initializes the prefix that will be used to make sure read names are (hopefully) unique
   private val actualReadNamePrefix = readNamePrefix.getOrElse {
-    header.getReadGroups.map(rg => Option(rg.getLibrary).getOrElse(rg.getReadGroupId)).toSet.toList.sorted.mkString("|")
+    val ids = header.getReadGroups.map(rg => Option(rg.getLibrary).getOrElse(rg.getReadGroupId)).toSet.toList.sorted
```

Comment:
`toSet.toList.sorted` works but what about using `distinct` instead of `toSet`?

```
val ids = header.getReadGroups.map(rg => Option(rg.getLibrary).getOrElse(rg.getReadGroupId)).sorted.distinct
````